curriculum/challenges/english/01-responsive-web-design/applied-visual-design/adjust-the-background-color-property-of-text.md

```diff
@@ -26,7 +26,7 @@ Also for the <code>h4</code>, remove the <code>height</code> property and add <c
 ```yml
 tests:
   - text: Your code should add a <code>background-color</code> property to the <code>h4</code> element set to <code>rgba(45, 45, 45, 0.1)</code>.
-    testString: assert(code.match(/(background-color|background):\s*?rgba\(\s*?45\s*?,\s*?45\s*?,\s*?45\s*?,\s*?0?\.1\s*?\);/gi));
+    testString: assert(code.match(/(background-color|background):\s*?rgba\(\s*?45\s*?,\s*?45\s*?,\s*?45\s*?,\s*?0?\.1\s*?\)(;?\s*?}|;\s*?)/gi));
```

Comment:
```suggestion
    testString: assert(code.match(/(background-color|background):\s*?rgba\(\s*?45\s*?,\s*?45\s*?,\s*?45\s*?,\s*?0?\.1\s*?\);?/gi));
```

This seems a bit more DRY and still worked locally. Is there a particular reason for checking a whitespace after the semicolon that I might be missing?

Reply:
This will cause the tests to pass on false positives. The check must be made that `;` is only optional if the next object is a `}`

Reply:
Ah - thanks for the clarification! 🙂